hazelcast-integration/spring-data-jpa-hazelcast-migration/after/after-hz-main/src/main/resources/hazelcast.xml

```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hazelcast
-        xsi:schemaLocation="http://www.hazelcast.com/schema/config
-        http://www.hazelcast.com/schema/config/hazelcast-config-3.11.xsd"
-        xmlns="http://www.hazelcast.com/schema/config"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<hazelcast xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Seems to work with 4.0 although couldn't run it, probably needs a running database.